TGX Files/Data/Localized/STRINGS_10.INI

```diff
@@ -153,7 +153,7 @@
 6158	Golemancy ProperName	9	Golemancy	Golemancy	Technology ProperName
 6159	Golemancy Description	205	The use of powerful elemental magic has imbued a semblance of life into the very earth, creating mindless warriors that feel no pain. Those wrought from khaldunite-bearing stone are particularly dangerous.	The use of powerful elemental magic has imbued a semblance of life into the very earth, creating mindless warriors that feel no pain. Those wrought from khaldunite-bearing stone are particularly dangerous.	Technology Description
 6160	Conjurors' Academy ProperName	18	Conjurors' Academy	Conjurors' Academy	Technology ProperName
-6161	Conjurors' Academy Description	331	Your kingdom's mages have delved into the dark side of summoning, and these fearsome Conjurors have learned the secret of calling forth creatures of Shadow. Their arcane research has led to new techniques that produce stronger and healthier mystical beasts; All summoned creatures have more health and deal more damage than normal.	Your kingdom's mages have delved into the dark side of summoning, and these fearsome Conjurors have learned the secret of calling forth creatures of Shadow. Their arcane research has led to new techniques that produce stronger and healthier mystical beasts; All summoned creatures have more health and deal more damage than normal.	Technology Description
+6161	Conjurors' Academy Description	331	Your kingdom's mages have delved into the dark side of summoning, and these fearsome Conjurors have learned the secret of calling forth creatures of Shadow. Their arcane research has led to new techniques that produce stronger and healthier mystical beasts; all summoned creatures have more health and deal more damage than normal.	Your kingdom's mages have delved into the dark side of summoning, and these fearsome Conjurors have learned the secret of calling forth creatures of Shadow. Their arcane research has led to new techniques that produce stronger and healthier mystical beasts; all summoned creatures have more health and deal more damage than normal.	Technology Description
```

Comment:
Doesn't quite look right in game. We're going to need to reword the description to include the new effects.

Reply:
Happy to workshop this, currently it's just a mashup of the first half of the Conjuror unit description and the 2nd half of the Summoning Scrolls description. What are the new effects you're referring to?